packages/flutter/lib/src/cupertino/picker.dart

```diff
@@ -258,6 +263,15 @@ class _CupertinoPickerState extends State<CupertinoPicker> {
     widget.onSelectedItemChanged?.call(index);
   }
 
+  void _handleChildTap(int index, FixedExtentScrollController controller) {
+    // Animation curve and duration eyeballed from an iOS 17.5 simulator.
```

Comment:
Can we move the curve & duration to the Cupertino constants file? (along with the comment about eyeballing the values on a Simulator)

Reply:
@navaronbracke  I do wonder if making these constants public increases our maintenance burden in the future without providing any considerable gain.